Dockerfile

```diff
@@ -14,11 +13,21 @@ ENV NODE_ENV=production
 COPY . .
 RUN npm run build
 
-# The same image but now only install the production dependencies as the frontend is already built using vite in the build-stage
-FROM gcr.io/distroless/nodejs22-debian12@sha256:b765815eafacee5222bfa50179028f41dd8c642b68ad68ec4e6922d3b1ff2710 AS production
+# Remove dev dependencies
+RUN npm prune --omit=dev
 
+
+# PRODUCTION STAGE
+FROM gcr.io/distroless/nodejs24-debian12 AS production
```

Comment:
please use hash pinning as a good practice

Reply:
I wanted to be lazy and wait for renovate bot to find the correct hashes, but you’re right, better this way :)